tensorflow/python/keras/applications/mobilenet_v2.py

```diff
@@ -500,14 +500,42 @@ def _make_divisible(v, divisor, min_value=None):
 
 @keras_export('keras.applications.mobilenet_v2.preprocess_input')
 def preprocess_input(x, data_format=None):
-  return imagenet_utils.preprocess_input(x, data_format=data_format, mode='tf')
+  """Preprocesses a numpy array encoding a batch of images.
+
+  Arguments
+    x: A 4D numpy array consists of RGB values within [0, 255].
+
+  Returns
+    Preprocessed array.
+
+  Raises
+    ValueError: In case of unknown `data_format` argument.
+  """
```

Comment:
Why is this needed? We are overriding the doc below right?

Same for decode_predictions

Reply:
My bad, you're right. I just pushed a commit removing the unnecessary docs. Thank you for the review!